server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -74,13 +78,18 @@ protected Map<String, Long> indexData(int numberOfIterations, boolean invokeFlus
             indexingStats.put(MAX_SEQ_NO_REFRESHED_OR_FLUSHED + "-shard-" + shardId, maxSeqNoRefreshedOrFlushed);
             refreshedOrFlushedOperations = totalOperations;
             int numberOfOperations = randomIntBetween(20, 50);
-            for (int j = 0; j < numberOfOperations; j++) {
-                IndexResponse response = indexSingleDoc(index);
-                maxSeqNo = response.getSeqNo();
-                shardId = response.getShardId().id();
-                indexingStats.put(MAX_SEQ_NO_TOTAL + "-shard-" + shardId, maxSeqNo);
+            int numberOfBulk = randomIntBetween(1, 5);
```

Comment:
Maybe mention in a comment on the test or in the PR/issue that we switched to bulk indexing to reduce number of translog syncs.

Reply:
Its just not the translog sync , but in general number of indexing requests. I have mentioned it already in the PR .